test/board_test.py

```diff
@@ -1,33 +1,63 @@
-import pytest
+import pytest # type: ignore
 from src.board import Board
+from src.coordinate import Coordinate
+from src.snake import Snake
 
 class TestBoard:
     def get_board(self):
+        width = 15
         snakes = [{
-            "coords": [{2, 2}, {2,3}, {3,3}],
+            "coords": [(2, 2), (2,3), (3,3)],
             "health": 10,
             "id": "asdf1234"
         }]
         foods = [
-            {10, 5},
-            {6, 12}
+            (10, 5),
+            (6, 12)
         ]
         return Board({
             "snakes" : snakes,
-            "food"  : foods
+            "food"  : foods,
+            "width" : width
```

Comment:
The problem with lining up colons like this without IDE supporting the code style is that it's annoying to maintain. For example, I'm pretty sure food's and width's colons are one space off from snake's.

- [ ] Don't line up colons, just use `key: value`